code/pizzabot.js

```diff
@@ -1,7 +1,64 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+
+const pizzaPrice = 80;
+
+const checkOrderName = orderName => {
+  if (
+    orderName === vegetarian ||
```

Comment:
This will work as well! 
```suggestion
    orderName == vegetarian ||
```